tests/test_version_req.rs

```diff
@@ -413,3 +413,18 @@ fn test_leading_digit_in_pre_and_build() {
         req(&format!("{} 1.2.3-1a-1a+1a-1a-1a", op));
     }
 }
+
+#[test]
+fn test_wildcard_and_another() {
+    let err = req_err("*, 0.20.0-any");
+    assert_to_string(
+        err,
+        "wildcard req (*) must be the only comparator in the version req",
+    );
+
+    let err = req_err("0.20.0-any, *");
+    assert_to_string(
+        err,
+        "unexpected character '*' while parsing major version number",
```

Comment:
This is a nit for sure, but I think this error could be a lot clearer. Seeing this I'd be very confused and look at how I typed `0.20.0-any` as opposed to considering that `, *` may not be legal.